tests/unit/amo/reducers/test_home.js

```diff
@@ -70,15 +70,23 @@ describe(__filename, () => {
 
       const homeState = store.getState().home;
 
-      expect(homeState.resultsLoaded).toEqual(true);
       expect(homeState.collections).toHaveLength(1);
       expect(homeState.collections[0]).toEqual(null);
-      expect(homeState.featuredExtensions).toEqual([
-        createInternalAddon(fakeAddon),
-      ]);
-      expect(homeState.featuredThemes).toEqual([
-        createInternalAddon(fakeTheme),
-      ]);
+    });
+
+    it('loads an empty array if featured themes is null', () => {
+      const { store } = dispatchClientMetadata();
+
+      store.dispatch(
+        loadHomeAddons({
+          collections: [null],
```

Comment:
Is that realistic? Could it be `collections: []` instead?

Reply:
The saga will load a `null` into the `collections` array if there is a problem fetching a collection, as is evidenced in the test above, but I guess we don't need to reuse that here. So yes, I can just make it `collections: []`.